include/odp/api/spec/timer.h

```diff
@@ -135,6 +135,21 @@ typedef struct {
  * Timer capability
  */
 typedef struct {
+	/** Common for all clock sources */
+	struct {
+		/** Maximum number of timer pools of any clock source */
+		uint32_t max_pools;
+	} common;
+
+	/** Maximum number of timer pools */
+	uint32_t max_pools;
```

Comment:
What's the difference between `max_pools` and `common.max_pools` here? This seems confusing.

Reply:
`max_pools` is for the clock source `odp_timer_capability()` was called with. `common.max_pools` is for all clock sources combined.

Reply:
Then wouldn't something like `this_clock_max_pools` and `all_clocks_max_pools` be a bit less confusing? If variable needs an explanation to understand it then perhaps a better name is needed.